Doc/whatsnew/index.rst

```diff
@@ -28,7 +28,7 @@ anyone wishing to stay up-to-date after a new release.
    2.1.rst
    2.0.rst
 
-The "Changelog" is a HTML version of the file :source:`Misc/NEWS` which
+The "Changelog" is a HTML version of the file :source:`Misc/NEWS.d` which
```

Comment:
NEWS.d is a directory with many files; could the text here be confusing or will people understand what is meant?

Reply:
Hmm.. good point. So what should this be changed into?

Reply:
I would suggest "is an HTML version of a file built from the contents of the directory tree..."  You could even link "file built" to blurb.

Reply:
Thanks I made a new PR: https://github.com/python/cpython/pull/3617